libraries/botbuilder-dialogs-adaptive/src/actions/dynamicBeginDialog.ts

```diff
@@ -6,12 +6,18 @@
  * Licensed under the MIT License.
  */
 
+import { ExpressionProperty } from 'adaptive-expressions';
 import { DialogContext } from 'botbuilder-dialogs';
 import { BeginDialog } from './beginDialog';
 
 export class DynamicBeginDialog extends BeginDialog {
-    protected bindOptions(dc: DialogContext, options: object): object {
-        // use overflow properties of deserialized object instead of the passed in option.
-        return super.bindOptions(dc, Object.assign({}, this));
+    protected bindOptions(dc: DialogContext, _options: object): object {
+        const options = {};
+        for (const key of Object.getOwnPropertyNames(this)) {
```

Comment:
This is not really related to trigger trees, right?

Reply:
No, but if I didn't make this change, the test will fail.